src/js/stores/search.js

```diff
@@ -153,10 +153,10 @@ define(function (require, exports, module) {
         handleExecute: function (id, itemID) {
             var idArray = itemID.split("-"),
```

Comment:
we don't use `id` at all in this function... is this intentional? can it be cleaned up?